live-examples/js-examples/expressions-logicaloperator.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">function addIfBothPositive(a, b) {
+  if (a > 0 && b > 0) {
+    return a + b;
+  }
+  return 0;
+}
+
+console.log(addIfBothPositive(2, 3));
+// expected output: 5
+
+console.log(addIfBothPositive(-2, 3));
+// expected output: 0
+</code>
```

Comment:
Nit: It's totally up to you, but it seems a shame to demonstrate only one of the operators. Maybe worth considering something like:

```
var a = 3;
var b = -2;

console.log(a > 0 && b > 0);
// expected output: false

console.log(a > 0 || b > 0);
// expected output: true

console.log(!(a > 0 || b > 0));
// expected output: false
```

Reply:
sure!